account_banking_sepa_direct_debit/views/report_sepa_direct_debit_mandate.xml

```diff
@@ -1,6 +1,10 @@
 <?xml version="1.0" encoding="utf-8" ?>
 <odoo>
     <template id="sepa_direct_debit_mandate_document">
+        <link
```

Comment:
Yo do not need load the style here.
The style sheet is not loaded because in manifest file the declaration of css is not right.

Change in manifest file assets_backend by report_assets_common

Reply:
Thanks for the correction.